angular-site/src/app/buttons-and-indicators/badge/badge-overview-example.html

```diff
@@ -0,0 +1,27 @@
+<h2 class="example-h2">Badge</h2>
+<p>
+  <span matBadge="4" matBadgeOverlap="false">Text with a badge</span>
+</p>
+
+<p>
+  Button with a badge on the left
+  <button mat-raised-button color="primary"
+          matBadge="8" matBadgePosition="before" matBadgeColor="accent">
+    Action
+  </button>
+</p>
+
+<p>
+  Icon with a badge
+  <mat-icon matBadge="15" matBadgeColor="warn">home</mat-icon>
+  <!-- Include text description of the icon's meaning for screen-readers -->
+  <span class="cdk-visually-hidden">
+      Example with a home icon with overlaid badge showing the number 15
+    </span>
+</p>
+
+
+
+<!-- Copyright 2019 Google LLC. All Rights Reserved.
```

Comment:
? do we must keep this info?

Reply:
You're right, it's unnecessary. Removed it